### DISCUSSION TITLE
$queryRaw does not query fields that connect the table to other tables

### DISCUSSION THREAD

**Discussion by Joostdeboer** [prisma/prisma#23455] [Q&A]
### Bug description

Due to some constraints with Prisma and nested queries, and us needing a very specific use-case, we decided to create an SQL query using $queryRaw, but ran into an issue that seems like a bug:

Using an SQL statement such as `SELECT * From Project`, we only obtain data from the Project that corresponds to basic fields in Prisma, but not any of the fields that are connections to other models.

### How to reproduce

Make a model with a relation to another model. Query the initial model using `await prisma.$queryRaw` using the statement: `SELECT * From Project` (Where "Project" can be replaced by your model name). You will not see the relation included in the returned result.

### Expected behavior

The offerEvents array is included in the query results:

```
[
  ...,
  {
    id: 'projectId1',
    name: 'My Project',
    createdAt: 2024-02-21T08:53:02.039Z,
    updatedAt: 2024-02-21T08:53:02.094Z,
    removedAt: null,
    offerEvents: [
      { ... }
    ],
  },
  {
    id: 'projectId2',
    name: 'project2',
    createdAt: 2024-02-23T10:49:58.613Z,
    updatedAt: 2024-02-23T10:50:05.320Z,
    removedAt: null,
    offerEvents: [
      { ... }
    ],
  },
  ...
]
```

### Prisma information

Here are the models we use, simplified:

```
model Project {
  id             String          @id @default(uuid())
  name           String?
  createdAt      DateTime        @default(now())
  updatedAt      DateTime        @updatedAt
  removedAt      DateTime?
  offerEvents    OfferEvent[]
}

model OfferEvent {
  id               String          @id @default(uuid())
  createdAt        DateTime        @default(now())
  project          Project         @relation(fields: [projectId], references: [id])
  projectId        String
}
```

```ts
const result = await this._prismaClient.$queryRaw`
    SELECT * FROM "Project"
`;
console.log({ result });
```

The result of the above query gives data such as the following, where, as can be seen, the list of offerEvents is not included.

```
[
  ...,
  {
    id: 'projectId1',
    name: 'My Project',
    createdAt: 2024-02-21T08:53:02.039Z,
    updatedAt: 2024-02-21T08:53:02.094Z,
    removedAt: null,
  },
  {
    id: 'projectId2',
    name: 'project2',
    createdAt: 2024-02-23T10:49:58.613Z,
    updatedAt: 2024-02-23T10:50:05.320Z,
    removedAt: null,
  },
  ...
]
```

### Environment & setup

-OS: macOS
Database: SQL Server
Node.js version: v18.19.0

### Prisma Version

```
5.10.1
```

**Comment by janpio** [comment]
The `offerEvents` data is not included in the `Project` table, so just selecting from that table can not return that data. You will have to join the other table manually in your SQL query via the `projectId` foreign key in `OfferEvent` to be able to return the list of events as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
